### PR TITLE
Valve weights

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: node/src/chain_spec/neumann.rs [hunk 1]
```diff
@@ -5,14 +5,14 @@ use sc_service::ChainType;
 use sc_telemetry::TelemetryEndpoints;
 use sp_core::{crypto::UncheckedInto, sr25519};
 
+use super::TELEMETRY_URL;
```
Comment (irsal) [inline node/src/chain_spec/neumann.rs:8]: thx for the lint

File: pallets/valve/src/benchmarking.rs [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// This file is part of OAK Blockchain.
+
+// Copyright (C) 2021 OAK Network
```
Comment (irsal) [inline pallets/valve/src/benchmarking.rs:3]: 2022

File: pallets/valve/src/weights.rs [hunk 3]
```diff
@@ -0,0 +1,105 @@
+
+//! Autogenerated weights for `pallet_valve`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-03-02, STEPS: `20`, REPEAT: 64, LOW RANGE: `[]`, HIGH RANGE: `[]`
```
Comment (irsal) [inline pallets/valve/src/weights.rs:5]: how frequently do we have to regenerate these weights?

**Review by irsal — APPROVED** [review]

**Comment by irsal** [comment]
merge conf, probs need to update 0.9.17 client as well

File: pallets/valve/src/weights.rs (hunk 3 above)
Comment (rhuttman) [reply pallets/valve/src/weights.rs:5]: Whenever we change something that could change the weights. That could be changing the logic in a function or changing pallet config params

File: pallets/valve/src/benchmarking.rs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+	}: close_pallet_gate(RawOrigin::Root, b"System".to_vec())
+
+
+	open_pallet_gate {
+		let pallet_name = b"System".to_vec();
+		ClosedPallets::<T>::insert(pallet_name.clone(), ());
+	}: _(RawOrigin::Root, pallet_name)
+
+	open_pallet_gates {
+		ClosedPallets::<T>::insert(b"System".to_vec(), ());
```
Comment (justinzhou93) [inline pallets/valve/src/benchmarking.rs:48]: if i remember correctly, you mentioned there are WAYYY more gates than this. shouldn't we open all of the gates that exist to make sure we get the correct amount? or alternatively use iterator `v`?

**Review by justinzhou93 — APPROVED** [review]
1 question, otherwise lgtm

**Pull request merged**